openshift/template.yaml

```diff
@@ -116,7 +116,7 @@ parameters:
 - name: ES_STEP_INDEX
   value: "steps"
 - name: ES_JOB_INDEX
-  value: "jobs"
+  value: "jobs-*"
```

Comment:
why this change?

Reply:
Just to say it explicitly, you should always have as much context as needed in the PR description and commit message instead of saying what you're doing in the PR

For example:
```
Searching `jobs` index does not yield any result, because the created indexes to the jobs are of the pattern: `jobs-<date>`. For example `jobs-2022-11-21`.

With current code we're getting the following result:

Querying elasticsearch with the query ...
Got response: []
```

Reply:
:+1: next time add description to the PR please :)
Also, did you look into [aliases](https://opensearch.org/docs/1.2/opensearch/index-alias/)?
They might be a better fit for this purpose

Reply:
Interesting! and then you query the alias as an index?

Let's keep the wildcard for the moment, and see if we can leverage an alias later.

Reply:
@osherdp ok thanks I thought just making this quick patch in order to make it work, should have been more explict

Reply:
No worries
Just try for the next changes to be more explicit (and include the description also in the commit message itself)